mappings/net/minecraft/entity/passive/AxolotlEntity.mapping

```diff
@@ -1,4 +1,44 @@
 CLASS net/minecraft/class_5762 net/minecraft/entity/passive/AxolotlEntity
+	COMMENT Represents an axolotl, the cutest predator.
+	COMMENT
+	COMMENT <div class="fabric">
+	COMMENT <table border=1>
+	COMMENT <caption>Relations of this entity class with the codebase</caption>
+	COMMENT <tr>
+	COMMENT   <th><b>Relation</b></th><th><b>Class</b></th>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>Brain</td><td>{@link AxolotlBrain}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>EntityData</td><td>{@link AxolotlData}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>Look Control</td><td>{@link AxolotlLookControl}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>Model</td><td>{@link net.minecraft.client.render.entity.model.AxolotlEntityModel}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>Move Control</td><td>{@link AxolotlMoveControl}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>Renderer</td><td>{@link net.minecraft.client.render.entity.AxolotlEntityRenderer}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>Swim Navigation</td><td>{@link AxolotlSwimNavigation}</td>
+	COMMENT </tr>
+	COMMENT <tr>
+	COMMENT   <td>Tasks</td>
+	COMMENT   <td><ul>
+	COMMENT     <li>{@link net.minecraft.entity.ai.brain.task.PlayDeadTask}</li>
+	COMMENT     <li>{@link net.minecraft.entity.ai.brain.task.PlayDeadTimerTask}</li>
+	COMMENT   </ul></td>
+	COMMENT <tr>
+	COMMENT   <td>Variants</td><td>{@link Variant}</td>
```

Comment:
I doubt the necessity and usefulness of this particular table; they are quite visible and the tasks are discoverable in the brain class.

Reply:
They are useful as a "quick navigation" table, maybe also useful for the online javadoc as interactions with the codebase become less obvious.